TelepresenceBot/server-unit/core/mover.js

```diff
@@ -0,0 +1,23 @@
+function Mover(clientsAndRooms, emitter) {
+    var hasNoRooms = function(clientId) {
+        return !clientsAndRooms[clientId];
+    };
+
+    return {
+        moveIn: function(clientId, direction) {
+            if(hasNoRooms(clientId)) {
+                return;
+            }
+
+            var roomsClientIsIn = Object.keys(clientsAndRooms[clientId]);
+
+            for(var i = 0; i < roomsClientIsIn.length; i++) {
+                emitter.to(roomsClientIsIn[i]).emit('direction', direction);
```

Comment:
`roomsClientIsIn` are the keys for the rooms that a given client belongs too. Yes this is confusing, I'll look into making this easier to read 👍